vshard/storage/init.lua

```diff
@@ -1589,6 +1600,7 @@ local function bucket_recv_xc(bucket_id, from, data, opts)
     end
     local bucket_generation = M.bucket_generation
     local limit = consts.BUCKET_CHUNK_SIZE
+    local event_type = 'bucket_data_recv_txn'
```

Comment:
For the record, I am fine if you would store these in `vshard.consts`. We only shouldn't expose `vshard.consts` anywhere in the docs, but can use it for internals easily. Up to you.

Reply:
Consts looks cleaner, I suppose. Better to use them